### PR TITLE
The Lobby now has a Job Estimation list

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
When players ready, it will post their highest priority job as 

## Changelog

:cl:
add: You can now see player's jobs in the lobby.
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -122,12 +124,41 @@ SUBSYSTEM_DEF(ticker)
...
 					++totalPlayersReady
 					if(player.client?.holder)
 						++total_admins_ready
 
+					players[player.key] = player
+
+			sortTim(players)
+
+			if(CONFIG_GET(flag/show_job_estimation))
+				for(var/ckey as anything in players)
```
Comment (Zonespace27) [inline code/controllers/subsystem/ticker.dm:141]:
```suggestion
				for(var/ckey in players)
```

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -122,12 +124,41 @@ SUBSYSTEM_DEF(ticker)
...
+					var/mob/dead/new_player/player = players[ckey]
+					var/datum/preferences/prefs = player.client?.prefs
+					if(!prefs)
+						continue
+					if(!prefs.read_preference(/datum/preference/toggle/ready_job))
```
Comment (Zonespace27) [inline code/controllers/subsystem/ticker.dm:146]:
```suggestion
					if(!prefs?.read_preference(/datum/preference/toggle/ready_job))
```

File: code/controllers/subsystem/ticker.dm [hunk 3]
```diff
@@ -122,12 +124,41 @@ SUBSYSTEM_DEF(ticker)
...
+						continue
+
+					var/datum/job/J = prefs.get_highest_priority_job()
+					var/title = prefs.alt_job_titles?[J.title] || J.title
```
Comment (Zonespace27) [inline code/controllers/subsystem/ticker.dm:150]:
```suggestion
					var/datum/job/highest_job = prefs.get_highest_priority_job()
					var/title = prefs.alt_job_titles?[highest_job.title] || highest_job.title
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/controllers/subsystem/ticker.dm (hunk 2 above)
Comment (Kapu1178) [reply code/controllers/subsystem/ticker.dm:146]: Prefs is nullchecked the line above

File: code/controllers/subsystem/ticker.dm (hunk 3 above)
Comment (Kapu1178) [reply code/controllers/subsystem/ticker.dm:150]: No Gods, No Masters

viva la 1 letter

**Pull request merged**